src/views/health-checks.vue

```diff
@@ -0,0 +1,83 @@
+<template>
+    <div class="tab">
+        <div v-if="compatible">
+            <h1>Health checks</h1>
+            <p>This is a baby tab. Currently it only checks if all the Custom Elements you're using are imported. Soon we'll add more health checks you can use for your apps.
+            <p><button v-on:click="checkElementsHealth">Run checks</button></p>
+            <hr />
+            <table>
+                <tr>
+                    <th>CE name</th>
+                    <th>Is imported</th>
+                </tr>
+                <tr v-for="CE in customElementsHealth" v-bind:key="CE.name" v-bind:class="[CE.isDefined ? 'ok-bg' : 'error-bg']">
+                    <td>{{CE.name}}</td>
+                    <td>{{CE.isDefined ? 'Yes': 'No'}}</td>
+                </tr>
+            </table>
+        </div>
+        <div v-if="!compatible">
+            <p>Sorry. Health checks are only available for WebComponents V1 and Polymer 2 (Starcounter v2.4)</p>
+        </div>
+    </div>
+</template>
+
+<script>
+import findUndefinedElements from '../utils/findUndefinedCustomElements';
+
+var currentImports = [];
+
+export default {
+  name: 'health-checks',
+  props: ['overlay'],
+  data() {
+    return {
+      compatible:
+        this.getCurrentWindow().Polymer &&
+        this.getCurrentWindow().Polymer.version.startsWith('2'), // only works with Polymer 2
```

Comment:
The recommended way to test for Polymer 2+ is to check if `Polymer.Element` is defined.

Source: https://www.polymer-project.org/2.0/docs/devguide/hybrid-elements